### ISSUE TITLE
Discussion: Milk dispenser

### ISSUE THREAD

**Issue by Hinaichigo** [d3athrow/vgstation13#34833]
It seems to me that having milk available in the space brewery at roundstart is a bit excessive. You can make like 30 cheese wheels in the first 30 seconds of the round and have no issues feeding anyone. Also as someone else put it, it makes bartending too much of "chemist 2.0" (see #34769), where they just sit in one spot and make drinks without having to interact with other departments. And think of the cows. We all know the discussions and back and forth around this concept but I think it's worth experimenting with making this less immediately available, and I'm thinking of making a PR to do at least one of the following:

1. remove milk and soymilk from the space brewery
2. gate milk and soymilk behind machine upgrades
3. unmap space brewery from the bar at roundstart

I'd like to hear people's thoughts on this to find a consensus people can be mainly satisfied with. Other alternatives do exist like having a limited or otherwise resource-dependent supply of reagents in the space brewery (not just the slowly-refilling mechanic). Sometimes I think about having the chem dispensers require plasma to refill their bars, and things like that.

Keep in mind there is a large stockpile of milk in the kitchen at roundstart already.

[discussion]

**Comment by wotisjanitor** [comment]
You can't and shouldn't do all 3 of these steps. I lean toward removing the dispensers, but I think if they are removed they should not be nerfed further. Also, can't the chef already do this? He's the guy who starts with all the milk cartons and the universal enzymes.

**Comment by Hinaichigo** [comment]
The chef does have milk, but it's a limited supply of milk. 
Using it to make cheese prevents him from using it to make cake or other recipes.
If he wants more he then has to get a cow or order some more from cargo, which then can lead to more emergent gameplay and player-player interaction.

And from an aesthetic point of view, using up milk cartons has a lot more SOVL to it than using the dispenser.
The cartons have to go in the trash, and it's not just a direct reagent injection into the beaker or whatever.

> but I think if they are removed they should not be nerfed further

Agree with this on a practical if not idealistic level.

**Comment by Kurfursten** [comment]
I agree with you that the infinite milk dispenser is a little sad because it means there is no reason to keep your goat alive. I think it's interesting gameplay that rewards you for dealing with the goat's temper tantrums with being able to milk the goat. When you have a brewery for free most people just kill Einhorn right away. People still do regardless because botanists can synth milk as well! More reason to use biogenerator...

**Comment by SonixApache** [comment]
move to T2 imo

**Comment by Inorien** [comment]
> You can make like 30 cheese wheels in the first 30 seconds of the round and have no issues feeding anyone.

when was the last time you saw this happen when it wasnt deadpop

**Comment by Hinaichigo** [comment]
I do it myself sometimes as chef.

**Comment by kilozombie** [comment]
Here is the real reason I hate the milk in the dispenser: it makes the poutine ocean fucking trivial. Back before breweries were mapped, and also particularly before I knew you could get milk this way, there were two routes: very high potency plants and a biogenerator, and a couple dozen cows. If you were really, really crafty with either of these two routes, you could make a poutine ocean.

Roundstart milk is nowhere near enough to make a poutine ocean, but roundstart brewery makes the cheese trivial. Yes I've written like 100 words so far on the subject of the poutine ocean's """balance""" but it's too easy right now pls nerf. Not sure the best route.

**Comment by pedr69** [comment]
milking femstatics when
